Iris/ic/CompilerRunner.cs

```diff
@@ -101,6 +105,12 @@ private static bool TryParseArgs(string[] args, out string sourceFile, out Compi
             if (sourceFile == null)
                 return false;
 
+
+#if NETCOREAPP
```

Comment:
If I'm understanding the .csproj correctly, aren't we always building "ic" with .NET 5 now?  If so why do we have conditional compilation for .NET Core?

Reply:
I left that in in case anyone wanted to modify the project to target desktop. But you are correct that it will always be used. I think I will leave a comment.